Products/mcdutils/sessiondata.py

```diff
@@ -60,13 +60,15 @@ def addItemsToSession(self):
         """ Add key value pairs from 'items' textarea to the session.
         """
         request = self.REQUEST
-        items = request.form.get('items', ())
+        items = request.form.get('items', [])
         session = request['SESSION']
 
         before = len(session)
         count = len(items)
 
         for line in items:
+            if not isinstance(line,bytes):
+                line = line.encode()
```

Comment:
Not passing an encoding means you rely on some default that may change over time. Please pass something explicitly instead, like 'UTF-8'.